app/javascript/api/types/rails/listings/BaseRailsListing.d.ts

```diff
@@ -40,6 +40,7 @@ export type ListingLotteryPreference = {
   Order: number
   Total_Submitted_Apps: number
   attributes: ListingAttributes
+  PDF_URL?: string
```

Comment:
double checking because i don't recall seeing this before. just want to make sure this isn't `pdfUrl`?

Reply:
disregard. i see it now!